File: Assignment 7/program3.c
#include<stdio.h>

void Display(int iNo)
{
    int iCnt=0;
    
    for(iCnt=-4;iCnt<=iNo;iCnt++)
    {
        printf("%d\t",iCnt);
    }

}
int main()
{
    int iValue=0;
    printf("Enter the Number You want to Print:");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

File: Assignment 26/program3.java
import java.util.*;

class Pattern
{ 
    public void Display(int iNO)
    {
        int iCnt=0;
        

        for(iCnt=1;iCnt<=iNO;iCnt++)
        {
            System.out.print(iCnt+ " * ");
        
        }

    }

}

class program3
{
    public static void main(String A[])
    { 
        int iValue=0;

        Scanner sobj=new Scanner(System.in);
        System.out.println("Enetr Number N:");
        iValue=sobj.nextInt();

        Pattern pobj=new Pattern();
        pobj.Display(iValue);

    }
}